.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,11 @@
+# Checklist
+
+- [ ] Updated the `pubspec.yaml` version in accordance with [Semantic Versioning](https://semver.org/)
+- [ ] Updated the `CHANGELOG.md` to reflect changes
+- Modified Rules. Keeping in mind that the Dart and Flutter rules should closely match:
+  - Dart
+    - [ ] Updated; OR
+    - [ ] Not Applicable
+  - Flutter
+    - [ ] Updated; OR
+    - [ ] Not Applicable
```

Comment:
This currently doesn't work with the `jcli git-pr` utility.

Thoughts on making it a feature to add?

- If `.github/PULL_REQUEST_TEMPLATE.md`, Prepend `<Current Branch>` to the content and use that as the PR body. Otherwise use `<Current Branch>`?

Reply:
https://github.com/jesims/jcli/issues/13

Reply:
https://github.com/jesims/jcli/pull/14